repo/tuf_on_ci/online_sign.py

```diff
@@ -29,10 +29,13 @@ def _git(cmd: list[str]) -> subprocess.CompletedProcess:
 @click.command()  # type: ignore[arg-type]
 @click.option("-v", "--verbose", count=True, default=0)
 @click.option("--push/--no-push", default=False)
-def online_sign(verbose: int, push: bool) -> None:
+@click.option("--role", default="")
+def online_sign(verbose: int, push: bool, role: str) -> None:
```

Comment:
I left a higher level comment on the issue: it kind of looks like doing the signing in the repo code is correct but combining the current online-sign and the new feature make this function more complicated but does not have synergies: a new top-level CLI command might make sense